loggers/default.go

```diff
@@ -0,0 +1,155 @@
+package loggers
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"strconv"
+	"strings"
+)
+
+type defaultEvent struct {
+	level  LogLevel
+	logger *log.Logger
+	entry  []byte
+}
+
+var boolMap = map[bool]string{
+	true:  "true",
+	false: "false",
+}
+
+func (e *defaultEvent) Msg(msg string) {
+	if len(msg) == 0 {
+		return
+	}
+
+	s := strings.Builder{}
+	s.WriteString("[")
+	s.WriteString(e.level.String())
+	s.WriteString("] ")
+	s.WriteString(msg)
+	s.Write(e.entry)
+
+	e.logger.Println(s.String())
+}
+
+func (e *defaultEvent) Str(key, val string) Event {
+	e.entry = append(e.entry, ' ')
+	e.entry = append(e.entry, key...)
+	e.entry = append(e.entry, '=')
+	e.entry = append(e.entry, val...)
+	return e
+}
+
+func (e *defaultEvent) Err(err error) Event {
+	if err == nil {
+		return e
+	}
+
+	e.entry = append(e.entry, " error="...)
+	e.entry = append(e.entry, err.Error()...)
+	return e
+}
+
+func (e *defaultEvent) Bool(key string, b bool) Event {
+	e.entry = append(e.entry, ' ')
+	e.entry = append(e.entry, key...)
+	e.entry = append(e.entry, '=')
+	e.entry = append(e.entry, boolMap[b]...)
+	return e
+}
+
+func (e *defaultEvent) Int(key string, i int) Event {
+	e.entry = append(e.entry, ' ')
+	e.entry = append(e.entry, key...)
+	e.entry = append(e.entry, '=')
+	e.entry = append(e.entry, strconv.Itoa(i)...)
+	return e
+}
+
+func (e *defaultEvent) Uint(key string, i uint) Event {
+	e.entry = append(e.entry, ' ')
+	e.entry = append(e.entry, key...)
+	e.entry = append(e.entry, '=')
+	e.entry = append(e.entry, strconv.Itoa(int(i))...)
+	return e
+}
+
+func (e *defaultEvent) Stringer(key string, val fmt.Stringer) Event {
+	e.entry = append(e.entry, ' ')
+	e.entry = append(e.entry, key...)
+	e.entry = append(e.entry, '=')
+	e.entry = append(e.entry, val.String()...)
+	return e
+}
+
+type defaultLogger struct {
```

Comment:
This is the default implementation based on standard `log` package cc @manojgop @anuraaga @ns-sundar